### PR TITLE
Luvjain/protectroute

### PR DESCRIPTION
**Pull request by LuvJain**

### CODE REVIEW

**Comment by LuvJain** [comment]
Should have been a separate branch, sorry
Added validation
Table updates
Some WIP for personal routes

File: server/init.sql [hunk 1]
```diff
@@ -124,7 +124,7 @@ ALTER TABLE ONLY public.family_events ALTER COLUMN id SET DEFAULT nextval('publi
 COPY public.events (id, event_name, venue, created_at, updated_at) FROM stdin;
 1	Ceremony	Hyatt Chesapeake Bay Manor Lawn	2021-05-17 00:00:00	2021-05-17 00:00:00
 2	Reception	Hyatt Chesapeake Bay Chesapeake Ballroom	2021-05-17 00:00:00	2021-05-17 00:00:00
-3	Friday Night	lol	2021-05-17 00:00:00	2021-05-17 00:00:00
+3	Meet & Greet + Sangeet	Hyatt Chesapeake Bay Choptank Ballroom	2021-05-17 00:00:00	2021-05-17 00:00:00
```
Comment (krishisharma45) [inline server/init.sql]: Change to Sangeet.

Comment (krishisharma45) [reply server/init.sql]: Also, lowercase: "sangeet"

File: server/init.sql [hunk 2]
```diff
@@ -124,7 +124,7 @@ ALTER TABLE ONLY public.family_events ALTER COLUMN id SET DEFAULT nextval('publi
 COPY public.events (id, event_name, venue, created_at, updated_at) FROM stdin;
 1	Ceremony	Hyatt Chesapeake Bay Manor Lawn	2021-05-17 00:00:00	2021-05-17 00:00:00
 2	Reception	Hyatt Chesapeake Bay Chesapeake Ballroom	2021-05-17 00:00:00	2021-05-17 00:00:00
```
Comment (krishisharma45) [inline server/init.sql]: lowercase: "reception"

File: server/init.sql [hunk 3]
```diff
@@ -124,7 +124,7 @@ ALTER TABLE ONLY public.family_events ALTER COLUMN id SET DEFAULT nextval('publi
 COPY public.events (id, event_name, venue, created_at, updated_at) FROM stdin;
 1	Ceremony	Hyatt Chesapeake Bay Manor Lawn	2021-05-17 00:00:00	2021-05-17 00:00:00
```
Comment (krishisharma45) [inline server/init.sql]: lowercase: "ceremony"

**Pull request merged**